src/react/features/tools/update/UpdateView.js

```diff
@@ -17,7 +17,11 @@ const useUpdateView = (diory = {}) => {
   const updatedDiory = { ...diory, ...values }
   const dispatch = useDispatch()
   return {
-    fields: fields.map((field) => ({ ...field, value: updatedDiory[field.key] })),
+    fields: fields.map((field) => ({
+      ...field,
+      value: updatedDiory[field.key],
+      autoFocus: field.key === 'text',
```

Comment:
Autofocus logic should be in fields.js
  {
    key: 'text',
    label: 'Text',
    format: 'string',
   autoFocus: true,
  },